src/js/cart.js

```diff
@@ -3,13 +3,6 @@ import { getLocalStorage } from "./utils.mjs";
 function renderCartContents() {
   const cartItems = getLocalStorage("so-cart");
 
-  // Check if there are any items in the cart and if it's an array
-  if (!Array.isArray(cartItems) || cartItems.length === 0) {
-    document.querySelector(".product-list").innerHTML =
-      "<p>Your cart is empty.</p>";
-    return;
-  }
-
```

Comment:
@nengiamoni You're going to want to do the calculation here instead of below. And you're going to want to do something like:

```
 cartItems.forEach((product) => {
    const finalPrice = calculateFinalPrice(product).toFixed(2);

    const productHTML = `
      <li class="cart-card divider">
        <h2 class="card__name">${product.name}</h2>
        <p class="cart-card__price">Price: $${product.price}</p>
        <p class="cart-card__discount">Discount: ${product.discount}%</p>
        <p class="cart-card__final-price">Final Price: $${finalPrice}</p>
      </li>
    `;
    productListElement.innerHTML += productHTML;
  });
```

As it is now it is hardcoded to only show three products that are in the array below which is wrong.